tests/cli/fluvio_smoke_tests/e2e-basic.bats

```diff
@@ -129,6 +136,11 @@ teardown_file() {
     assert_success
     run timeout 15s "$FLUVIO_BIN" topic create "$TOPIC_NAME_12"
     assert_success
+
+    if [[ "$FLUVIO_CLI_RELEASE_CHANNEL" != "stable" || "$FLUVIO_CLUSTER_RELEASE_CHANNEL" != "stable" ]]; then
```

Comment:
I think that here it should be `and` instead of `or`
```suggestion
    if [[ "$FLUVIO_CLI_RELEASE_CHANNEL" != "stable" && "$FLUVIO_CLUSTER_RELEASE_CHANNEL" != "stable" ]]; then
```